app/app.js

```diff
@@ -1,12 +1,17 @@
-const express = require('express');
+const express = require("express");
 const app = express();
-const { getTopics } = require('./controllers/controllers');
+const { getTopics } = require("./controllers/controllers");
+const endpoints = require("../endpoints.json");
 
-app.get('/api/topics', getTopics);
+app.get("/api", (req, res) => {
+  res.status(200).json(endpoints);
```

Comment:
.json is fine to use but it is better practise to use .send as .send converts what is sent to JSON anyway but will only do so if what you are trying to send is valid